mantidproject/mantidimaging
# Adds purple mantid icons; Sets it to main window; sets app name to Mantid Imaging

- Adds purple mantid and unstable mantid icons
- Sets it to main window
- Sets app name to Mantid Imaging


### To test
- Start mantid imaging
- Check that the icon in the applications taskbar is the purple mantid
- Check that the name of the app is not `__main__.py`

Does **NOT** currently distinguish which one it is. This will be done in https://github.com/mantidproject/mantidimaging/issues/638

Fixes #288
